gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -46,7 +46,17 @@ var gpii = fluid.registerNamespace("gpii");
             }
         },
         members: {
-            sessionIndex: {} // map of userToken to session component member name, managed by gpii.lifecycleManager.sessionIndexer
+            sessionIndex: {}, // map of userToken to session component member name, managed by gpii.lifecycleManager.sessionIndexer
+            queue: [], // queue for high-level lifecycle manager tasks
```

Comment:
Should itemise clearly which are the current tasks we expect to support, and what the contract is on being a task - e.g. function accepting (1 or more arguments) and returning (promise) etc.